### PR TITLE
Fatal if project's version is not found + fatalMsg() improvement

### PR DESCRIPTION
**Pull request by sdurrheimer**
Ref. #7

### CODE REVIEW

File: cmd/crossbuild_tarballs.go [hunk 1]
```diff
@@ -53,7 +54,8 @@ func runCrossbuildTarballs() {
 			os.Setenv("GOOS", platform[0])
 			os.Setenv("GOARCH", platform[1])
 		} else {
-			fatalMsg(nil, fmt.Sprintf("Bad .build/%s directory naming, should be <GOOS>-<GOARCH>", platform))
+			err := errors.New(fmt.Sprintf("Bad .build/%s directory naming, should be <GOOS>-<GOARCH>", platform))
```
Comment (grobie) [inline cmd/crossbuild_tarballs.go]: `fmt.Errorsf("bad .build/%s directory naming, should be <GOOS>-<GOARCH>", platform)`

File: cmd/promu.go [hunk 2]
```diff
@@ -108,7 +108,7 @@ func fatal(err error) {
 // fatalMsg prints a message and exit
 func fatalMsg(err error, msg string) {
 	if err != nil {
-		fmt.Println("!!", msg, err)
+		fmt.Println("!!", fmt.Sprintf("%s:", msg), err)
```
Comment (grobie) [inline cmd/promu.go]: `fmt.Printf("!! %s: %s\n", msg, err)`?

File: cmd/crossbuild_tarballs.go (hunk 1 above)
Comment (sdurrheimer) [inline cmd/crossbuild_tarballs.go]: Fixed

File: cmd/promu.go (hunk 2 above)
Comment (sdurrheimer) [inline cmd/promu.go]: Fixed

Comment (grobie) [inline cmd/promu.go]: Alternatively `fmt.Println("!!", msg, ":", err)`, if you prefer a format free one.

**Pull request merged**